docs/source/azure/examples/routetablewithsubnet-networking.rst

```diff
@@ -1,7 +1,7 @@
 .. _azure_examples_routetable_with_subnet:
 
-Find Route Tables With A Specific Subnet
-============================================
+Routes - Find route tables with a specific subnet
```

Comment:
`Routes` might be better replaced with `Networking`

Reply:
It is already in Networking group, so I think Routes makes sense because it groups examples inside a group.